client-coroutines/src/main/kotlin/io/provenance/client/coroutines/PbCoroutinesClient.kt

```diff
@@ -94,16 +94,11 @@ open class PbCoroutinesClient(
         val tx = mkTx {
             body = baseReq.body
             authInfo = baseReq.buildAuthInfo()
+            signaturesList.add(ByteString.EMPTY)
         }
 
-        val signatures = baseReq.buildSignDocBytesList(tx.authInfo.toByteString(), tx.body.toByteString())
```

Comment:
This change was made to mimic the change in `AbstractPbClient`, but this whole client needs a refresh.